### DISCUSSION TITLE
Pulsante TCP cambio stato

### DISCUSSION THREAD

**Discussion by denimassi** [indomus/forum#2702] [HUB: Home Assistant]
Buongiorno ho bisogno di un aiuto da parte vosta.
Ho creato in un Home assistant un Pulsante che tramite node red invia una stringa TCP ad un server DALI che controlla le luci, lo stesso server se interrogato in questo caso da Node red è in grado di rispondere con lo stato della lampada. 
Come faccio a cambiare lo stato della card in HA (spento/luce gialla accesso). Grazie

**Comment by snakuzzo** [comment]
Devi postare qui quello che hai fatto su Home Assistant e, mi raccomando, usando il markdown
